services/QuillLMS/lib/query_examples/admin_diagnostics/diagnostic-students-view.sql

```diff
@@ -1,19 +1,19 @@
         /*
-           Data Processed By Query: 1.73 GB
-           Bytes Billed For Query:  0.19 GB
-           Total Query Time:        1915 ms
-           Total Slot Time:         2377 ms
-           BI Engine Mode Used:     FULL_INPUT
-             BI Engine Code:          
-             BI Engine Message:       
+           Data Processed By Query: 2.49 GB
+           Bytes Billed For Query:  2.49 GB
+           Total Query Time:        2509 ms
+           Total Slot Time:         15991 ms
+           BI Engine Mode Used:     BI_ENGINE_DISABLED
+             BI Engine Code:          INPUT_TOO_LARGE
+             BI Engine Message:       Cannot broadcast table lms.active_user_names_view: number of rows 9636555 > supported limit of 8000000.
```

Comment:
This falling out of BI Engine is concerning.  I wonder if we've had data size growth pushing `active_user_names_view` out of BI Engine, or if this is just one of those cases where sometimes queries get this `INPUT_TOO_LARGE` error, but usually don't, and that this would be in BI Engine if it were re-run.

Reply:
Yeah, it's unclear what is happening. The query has been edited also, so that could have caused it too.